### PR TITLE
SEGrooms patch 1

### PR DESCRIPTION
**Pull request by SEGrooms**

### CODE REVIEW

File: video-categories/discrete-mathematics/video-list.json [hunk 1]
```diff
@@ -51,18 +51,72 @@
...
+      "video_author": "Shawn Grooms"
+    },
+    {
+      "title": "Conditional & Biconditional Examples",
+      "complete": false,
+      "script_author": "Shawn Grooms",
+      "video_author": "Shawn Grooms"
+    },
+    {
+     "title": "What Are Tautologies?,
```
Comment (atjonathan) [inline video-categories/discrete-mathematics/video-list.json]: ~~`"title": "What Are Tautologies?,`~~
`"title": "What Are Tautologies?",`

File: video-categories/discrete-mathematics/video-list.json [hunk 2]
```diff
@@ -51,18 +51,72 @@
...
+     "complete": false,
+     "script_author": "Shawn Grooms",
+     "video_author": "Shawn Grooms"
+    },
+    {
+     "title": "What Are Fallacies?,
```
Comment (atjonathan) [inline video-categories/discrete-mathematics/video-list.json]: ~~`"title": "What Are Fallacies?,`~~
`"title": "What Are Fallacies?",`

File: video-categories/discrete-mathematics/video-list.json [hunk 3]
```diff
@@ -51,18 +51,72 @@
...
+    },
+    {
+      "title": "Conditional & Biconditional Examples",
+      "complete": false,
+      "script_author": "Shawn Grooms",
+      "video_author": "Shawn Grooms"
+    },
+    {
+     "title": "What Are Tautologies?,
+     "complete": false,
```
Comment (SEGrooms) [inline video-categories/discrete-mathematics/video-list.json]: @atjonathan Do you know why "complete", "script_author", "Shawn Grooms", etc. are all in red?

Comment (atjonathan) [inline video-categories/discrete-mathematics/video-list.json]: @SEGrooms, you had some syntax errors, but they are fixed now :smiley: :tada:

**Pull request merged**